components/prism-tap.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.tap = {
+	fail: /(    )*not ok[^#\{\n]*/,
+	pass: /(    )*ok[^#\{\n]*/,
```

Comment:
You do not need to escape the `{` in the character class in those two regexps.
Also, we should probably include `\r` in the character class too, to handle the different line endings.